nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -518,14 +519,17 @@ function AddSubscriberDetails(props: DialogProps) {
             sub_profile: subscriber.dataPlan,
           },
         });
-        enqueueSnackbar('Subscriber(s) saved successfully', {
-          variant: 'success',
-        });
+        successCount++;
       } catch (e) {
         const errMsg = e.response?.data?.message ?? e.message ?? e;
         setError('error saving ' + subscriber.imsi + ' : ' + errMsg);
         return;
       }
+      if (successCount % 50 === 0 || successCount === subscribers.length) {
```

Comment:
Why are we checking if this is divisible by 50? If there's a reason for this pull it out and name it something meaningful so it isn't a magic number.

Reply:
We are just trying to batch the snackbar messages so that we don't flood the user with notifications when they are trying to upload 1000-10000 subscribers.  It would ideally be nice to replace it with some sort of a progress bar here, I don't know if some component is already available in fbcnms-ui.

Reply:
![Screen Shot 2021-01-30 at 8 00 11 AM](https://user-images.githubusercontent.com/8224854/106361154-3e6dc400-62d1-11eb-934e-68908e672d8e.png)